src/form-builder/PhoneField.tsx

```diff
@@ -17,13 +17,15 @@ const PhoneField = (props: PhoneProps): JSX.Element => {
     ...props,
     validate: chainValidations(props, [required, isValidPhone]),
   };
-  const [field, meta, helpers] = useField(withValidation as FieldHookConfig<string>);
+  const [field, meta, helpers] = useField(
+    withValidation as FieldHookConfig<string>
+  );
   const id = props.id || props.name;
   const value = field.value;
 
   return (
     <TextField
-      type='tel'
+      type="tel"
```

Comment:
I think now that the prettier linting is back we should keep change this to single quotes to keep things consistant.

Reply:
@bjmccotter7192 I think this change is from a merge / don't remember making this change